packages/picasso.js/src/core/chart-components/point/point.js

```diff
@@ -189,7 +196,9 @@ const component = {
     const limits = extend({}, SIZE_LIMITS, this.settings.settings.sizeLimits);
     const points = resolved.items;
     const pointSize = getPointSizeLimits(resolved.settings.x, resolved.settings.y, width, height, limits);
-    return createDisplayPoints(points, this.rect, pointSize, this.settings.shapeFn || this.symbol);
+    const shapeFn = this.settings.shapeFn || this.symbol;
+    const { errorShape } = this.settings;
```

Comment:
`errorShape` is extracted from the wrong "level" of setting. It should the `const { errorShape } = this.settings.settings;`

Reply:
To clarify, `this.settings` refers to generic component settings, those that are available to all components, like `data` or `dock`